docs/backend/setup/database-migrations.md

```diff
@@ -7,10 +7,11 @@ sidebar_position: 4
 If you need to change the database schema, you'll need to write a migration. These are the general steps you'll need to follow:
 
 1. [Generate a new migration file](#Creating-Migrations)
-1. [Add the new SQL to the generated file](#writing-migrations)
-1. [Set up your database](#Setup)
-1. [Run the migrations](#Running-Migrations)
-1. Test your new migration
+2. [Add the new SQL to the generated file](#writing-migrations)
+3. [Set up your database](#Setup)
+4. [Run the migrations](#Running-Migrations)
+5. [Rollback migrations locally](#Rollback-Migrations-Locally)
+6. Test your new migration
```

Comment:
Let's have Markdown create the numbered list for us rather than manually having to maintain it. 


```suggestion
1. [Add the new SQL to the generated file](#writing-migrations)
1. [Set up your database](#Setup)
1. [Run the migrations](#Running-Migrations)
1. [Rollback migrations locally](#Rollback-Migrations-Locally)
1. Test your new migration
```